packages/headless/src/modules/order/services/customer.service.ts

```diff
@@ -38,8 +38,10 @@ export class OrderCustomerService {
   }
 
   async reOrder(userId: string, body: ReOrderQuery): Promise<ReOrderResponse> {
-    const { ignoreInvalidItems, overWriteCart, orderId } = body;
-
+    let { ignoreInvalidItems, overWriteCart, orderId } = body;
+    if(!ignoreInvalidItems) ignoreInvalidItems = false//if ignoreInvalidItems is not provided, it is false by default
+    if(!overWriteCart) overWriteCart = false//if overwriteCart is not provided,it is false by default
```

Comment:
rather you could check similarly (using not sign) where you are checking (===false)